packages/app-mobile/components/screens/Note.tsx

```diff
@@ -807,8 +809,11 @@ class NoteScreenComponent extends BaseScreenComponent {
 
 	public async onAlarmDialogAccept(date: Date) {
 		const response = await checkPermissions(PermissionsAndroid.PERMISSIONS.POST_NOTIFICATIONS);
-		if (response !== PermissionsAndroid.RESULTS.GRANTED) {
-			logger.warn('POST_NOTIFICATION permission was not granted');
+
+		// The POST_NOTIFICATIONS permission isn't supported on Android API < 33.
+		// On earlier releases, notifications should work without this permission.
+		if (response !== PermissionsAndroid.RESULTS.GRANTED && getAndroidAPILevel() >= 33) {
+			logger.warn('POST_NOTIFICATIONS permission was not granted');
 			return;
```

Comment:
Currently, this only logs a warning and exits. It may make sense to instead show an alert dialog (e.g. "alarms won't go off") and not exit (continuing to set the due date).

Reply:
In shareHandler I've recently implemented similar logic except I only handle `if (response === PermissionsAndroid.RESULTS.DENIED)` and everything else, including `NEVER_ASK_AGAIN` is assumed to be permission granted. Would it make sense to have the same logic here?

Also to show a message, we can use `ToastAndroid` if you think that may be necessary

Reply:
>  Would it make sense to have the same logic here?

It could — users that selected "never ask again" in the UI (or through settings) might still want to be notified that alarms won't go off, however.

Reply:
> Also to show a message, we can use ToastAndroid if you think that may be necessary

As this is going to be part of a patch release, I don't plan to include that change.